### PR TITLE
KEYCLOAK-19431 Correcting link to SAML adapter

### PR DESCRIPTION
**Pull request by andymunro**
@hmlnarik This is just a fix to an incorrect link. Can you check it?

### CODE REVIEW

**Review by hmlnarik — CHANGES_REQUESTED** [review]
The fix does not correspond to KEYCLOAK-19431

**Comment by andymunro** [comment]
@hmlnarik, @pdrozd Can you verify this correction.

File: securing_apps/topics/overview/supported-platforms.adoc [hunk 1]
```diff
@@ -53,7 +53,10 @@ endif::[]
 * <<_saml_jboss_adapter,JBoss EAP>>
 ifeval::[{project_community}==true]
 * <<_saml_jboss_adapter,WildFly>>
-* <<_tomcat_adapter,Tomcat>>
+* <<_saml-tomcat-adapter,Tomcat>>
+endif::[]
+* <<java-servlet-filter-adapter,Servlet filter>>
```
Comment (hmlnarik) [inline securing_apps/topics/overview/supported-platforms.adoc]: Could the name of the anchor follow the same standard as other SAML adapters, e.g. `_saml-java-servlet-filter-adapter`?

Comment (andymunro) [reply securing_apps/topics/overview/supported-platforms.adoc]: Okay, that works.  Updated.

**Review by hmlnarik — APPROVED** [review]

**Pull request merged**